presto-main/src/main/java/io/prestosql/sql/planner/LocalExecutionPlanner.java

```diff
@@ -2144,14 +2146,17 @@ private DynamicFilterSourceOperatorFactory createDynamicFilterSourceOperatorFact
                         return new DynamicFilterSourceOperator.Channel(filterId, type, index);
                     })
                     .collect(Collectors.toList());
+            boolean isReplicatedJoin = node.getDistributionType()
```

Comment:
use `io.prestosql.operator.JoinUtils#isBuildSideReplicated` instead (it's used in `DynamicFilterService`)